staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -2873,6 +2873,10 @@ type PodSpec struct {
 	// This is an alpha feature and may change in the future.
 	// +optional
 	RuntimeClassName *string `json:"runtimeClassName,omitempty" protobuf:"bytes,29,opt,name=runtimeClassName"`
+	// List of user-initiated ephemeral containers to run in this pod.
+	// This field is alpha-level and is only honored by servers that enable the EphemeralContainers feature.
+	// +optional
+	EphemeralContainers []EphemeralContainer `json:"ephemeralContainers,omitempty" protobuf:"bytes,30,opt,name=ephemeralContainers"`
```

Comment:
@lavalamp should this list be keyed by name?

Reply:
Yes, it should be associative on the name field. (old (current) way of saying this is something mergeKeys=...)